portfolio/src/main/java/com/google/sps/model/Comment.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.model;
+
+import java.util.Date;
+
+public class Comment {
+  private long id;
+  private String content;
+  private String userName;
+  private Date time;
+  private long like;
+
+  public Comment(long id, String userName, String content, Date time, long like) {
+    this.id = id;
+    this.content = content;
+    this.userName = userName;
+    this.time = time;
+    this.like = like;
+  }
+
+  public void like() {
```

Comment:
do you really need this method?

Reply:
good catch! "like" is directly edited using datastore. method deleted